docker/docker_client.go

```diff
@@ -166,11 +178,13 @@ func dockerCertDir(sys *types.SystemContext, hostPort string) (string, error) {
 		hostCertDir     string
 		fullCertDirPath string
 	)
-	for _, systemPerHostCertDirPath := range systemPerHostCertDirPaths {
-		if sys != nil && sys.RootForImplicitAbsolutePaths != "" {
-			hostCertDir = filepath.Join(sys.RootForImplicitAbsolutePaths, systemPerHostCertDirPath)
+
+	systemPerHostCertDirs = append([]certPath{{path: filepath.Join(homedir.Get(), homeCertDir), legacyFormat: false}}, systemPerHostCertDirs...)
+	for _, systemPerHostCertDir := range systemPerHostCertDirs {
```

Comment:
These paths are no longer all `system`. Maybe `s/systemP/p/`, for both variables.

Reply:
fixed